app/src/main/res/menu/subscriptions.xml

```diff
@@ -32,4 +32,12 @@
                     android:title="5"/>
         </menu>
     </item>
+    <item
+        android:id="@+id/subscription_filter"
+        android:icon="?attr/ic_filter"
+        android:menuCategory="container"
+        android:title="@string/filter"
+        custom:showAsAction="always"/>
```

Comment:
Might be better to use `ifRoom`. Also, please move the definition up directly below the refresh icon because that is the order that is used in the menu.